v2.0/update.md

```diff
@@ -6,9 +6,11 @@ toc: false
 
 The `UPDATE` [statement](sql-statements.html) updates rows in a table.
 
+{{site.data.alerts.callout_danger}}If you update a row that contains a column referenced by a <a href="foreign-key.html">foreign key</a> and has an <a href="foreign-key.html#foreign-key-actions"><code>ON UPDATE</code> action</a>, all of the dependent rows will also be updated.{{site.data.alerts.end}}
```

Comment:
nit: `a foreign key and has an ON UPDATE action` > `a foreign key constraint with an ON UPDATE action`

Reply:
Done